### PR TITLE
Add copy icon

### PR DESCRIPTION
**Pull request by saralohr**
Screen Shot 2022-01-20 at 3 21 13 PM
Screen Shot 2022-01-20 at 3 21 22 PM
Screen Shot 2022-01-20 at 3 21 30 PM

### CODE REVIEW

**Review by analauhou — APPROVED** [review]

File: src/js/icons/calendar-today.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = {"metadata":{"xmlns":"http://www.w3.org/2000/svg","viewBox":"0 0 24 24","name":"calendar-today"},"source":"<path d=\"M0 0h24v24H0V0z\" fill=\"none\"/><path d=\"M20 3h-1V2c0-.55-.45-1-1-1s-1 .45-1 1v1H7V2c0-.55-.45-1-1-1s-1 .45-1 1v1H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm-1 18H5c-.55 0-1-.45-1-1V8h16v12c0 .55-.45 1-1 1z\"/>"}
```
Comment (BillyFigueroa) [inline src/js/icons/calendar-today.ts:1]: add a new line to end of file

File: src/js/icons/content-copy.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+module.exports = {"metadata":{"xmlns":"http://www.w3.org/2000/svg","viewBox":"0 0 24 24","name":"content-copy"},"source":"<rect fill=\"none\"/><path d=\"M15 20H5V7c0-.55-.45-1-1-1s-1 .45-1 1v13c0 1.1.9 2 2 2h10c.55 0 1-.45 1-1s-.45-1-1-1zm5-4V4c0-1.1-.9-2-2-2H9c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h9c1.1 0 2-.9 2-2zm-2 0H9V4h9v12z\"/>"}
```
Comment (BillyFigueroa) [inline src/js/icons/content-copy.ts:1]: add a new line to end of file

File: src/js/icons/more-horiz.ts [hunk 3]
```diff
@@ -1 +1 @@
-module.exports = {"metadata":{"xmlns":"http://www.w3.org/2000/svg","viewBox":"0 0 24 24","name":"more-horiz"},"source":"<path d=\"M0 0h24v24H0V0z\" fill=\"none\"/><path d=\"M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z\"/>"}
+module.exports = {"metadata":{"xmlns":"http://www.w3.org/2000/svg","viewBox":"0 0 24 24","name":"more-horiz"},"source":"<path d=\"M0 0h24v24H0V0z\" fill=\"none\"/><path d=\"M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z\"/>"}
```
Comment (BillyFigueroa) [inline src/js/icons/more-horiz.ts:1]: add a new line to end of file

File: src/js/icons/sell.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+module.exports = {"metadata":{"xmlns":"http://www.w3.org/2000/svg","viewBox":"0 0 24 24","name":"sell"},"source":"<path d=\"M0 0h24v24H0V0z\" fill=\"none\"/><path d=\"M21.41 11.41l-8.83-8.83c-.37-.37-.88-.58-1.41-.58H4c-1.1 0-2 .9-2 2v7.17c0 .53.21 1.04.59 1.41l8.83 8.83c.78.78 2.05.78 2.83 0l7.17-7.17c.78-.78.78-2.04-.01-2.83zM6.5 8C5.67 8 5 7.33 5 6.5S5.67 5 6.5 5 8 5.67 8 6.5 7.33 8 6.5 8z\"/>"}
```
Comment (BillyFigueroa) [inline src/js/icons/sell.ts:1]: add a new line to end of file

**Pull request merged**